app/controllers/users_projects_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersProjectsController < ApplicationController
+  def metrics
```

Comment:
for the future, it'd be great if our controllers are RESTful. e.g. for this case I'd have
```ruby
module Metrics
  class UsersProjectsController < ApplicationController
    def index
```

http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/